packages/devextreme/js/__internal/ui/chat/chat_message_list.ts

```diff
@@ -39,7 +44,9 @@ class MessageList extends Widget<MessageListOptions> {
 
     super._initMarkup();
 
+    this._renderScrollable();
     this._renderMessageListContent();
```

Comment:
The scrollable component needs to know what content it is working with during the initialization stage. If there is no content, it will incorrectly calculate the sizes of the simulated scrollbar. 
<img width="491" alt="image" src="https://github.com/user-attachments/assets/073e0ca0-6e86-47fb-a9fe-9b021df03101">
With the current approach, you add content without the component knowing about it. In this case, you need to call the update() method to inform the component about the content change.

Reply:
We have a task to integrate ScrollView. Now we have problems with Scrollable when invalidate is launched, so this option is not quite suitable. I used useNative: true to avoid the problems you describe